### PR TITLE
Issue Fixed #7080

### LINKED ISSUE #7080: Add Option button should shift downward for adding a new option in Bundle Product.
### Terms

- [X] Before you openning this issue, i have checked if the issue has already been reported.

### Bagisto Version(s) affected

MASTER

### Issue Description

When we create a **bundle product,** there we can see **Add option Button**, Once we add Options and we need to create a new Option, there we need to **scroll up** and then we select the add option button. It would be **better** if the **Add Option button** **shifts downward.**

### Preconditions

**1.** Simple product should be created so that to add the product to bundle options.
**2.** At least one option should be added, so that we can add new option.

### Steps to reproduce

1. Go to Admin -> Create a Bundle Product -> Add Options.
2. After adding First option, now create another option.
3.  Here we need to scroll up and then we select the add option button.

### Expected Result

It would be much better if the Add Option Button shifts downward after we add a new option. So that if we need to create another option so we would not scroll up for the button, in bottom we can see the button we can add directly.

### Actual Result

Please check the **video** for a better **understanding** of the **enhancement** what we can do in it.

**Video**

https://user-images.githubusercontent.com/92721837/199908038-f1e70d31-9080-434d-b694-6a480c81426c.mp4

### Screenshots

Please check the screenshot where we can add the button.

Add option

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by akhtarkhan1**
## Issue Reference
Issue Fixed #7080

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@akhtarkhan1 
This is **working fine**, please find the **screenshot** [Here](https://prnt.sc/24osCbuAY3-m)

**Review by Sanjaybhattwebkul — DISMISSED** [review]

File: packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/bundle-items.blade.php [hunk 1]
```diff
@@ -133,7 +133,7 @@
                         </tr>
                     </thead>
 
-                    <tbody>
+                    <tbody style="border-bottom: 1px solid #e8e8e8;">
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/bundle-items.blade.php:136]: Use class don't add direct styling.

File: packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/bundle-items.blade.php [hunk 2]
```diff
@@ -146,6 +146,11 @@
                             @onCheckProduct="checkProduct($event)">
                         </bundle-product-item>
 
+                        <tr v-if="!bundle_option_products.length">
+                            <td colspan="10">
+                                <p style="text-align:center;">{{ __('admin::app.catalog.products.no-product-added') }} </p>
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/bundle-items.blade.php:151]: Same here.

File: packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/bundle-items.blade.php [hunk 3]
```diff
@@ -15,17 +15,17 @@
...
-
             <bundle-option-item
                 v-for='(option, index) in options'
                 :option="option"
                 :key="index"
                 :index="index"
                 @onRemoveOption="removeOption($event)"
             ></bundle-option-item>
+
+            <button type="button" class="btn btn-md btn-primary" @click="addOption" style="margin-bottom: 20px;">
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/bundle-items.blade.php:26]: Same here.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by jitendra-webkul** [comment]
Resolve conflicts

**Comment by Sanjaybhattwebkul** [comment]
@akhtarkhan1 
Now, this issue is **fixed**. Please find the **screenshot** [Here](https://prnt.sc/H7X6xQhL5EDg)

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**